src/containers/Home.jsx

```diff
@@ -24,8 +24,8 @@ const styles = StyleSheet.create({
     marginBottom: 15,
     color: theme.colors.white
   },
-  link: {
-    ...theme.text.link
+  link_dark_bg: {
```

Comment:
I suggest naming link types by function rather than appearance, to leave room for both alternate themes and accessibility via non-visual styles. E.g. if we're using dark background to add emphasis, maybe something like `link_important`.